attributes/default.rb

```diff
@@ -21,6 +21,7 @@
 default['splunk']['web_port']       = '443'
 default['splunk']['ratelimit_kilobytessec'] = '2048'
 
+default['splunk']['change_splunk_auth'] = true
```

Comment:
Please rename the attribute to `node['splunk']['setup_auth']` to correspond to the recipe, making the intent more clear.